Sources/Sentry/SentryFileManager.m

```diff
@@ -324,6 +327,21 @@ - (void)deleteCrashedSession
     [self deleteSession:self.crashedSessionFilePath];
 }
 
+- (void)storeAbnormalSession:(SentrySession *)session
```

Comment:
`l`: Should we start marking parameters as `_Nonnull` to reduce nullability issues?

Reply:
It's nonnull cause of NS_ASSUME_NONNULL_BEGIN. Thanks for double checking.